functions/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
👍 Nice.  Makes sense that we don't need to keep the node modules in git because we should be rebuilding them with `npm i` after every pull as a matter of habit anyway.

... there's some hitch with ignoring tracked files, isn't there?  Like they remain tracked unless you specifically delete them or something?

Reply:
Here's the git command to untrack a file:

```
git rm --cached foo.txt 
```

You can also delete directly in the repo .